Snippets/Snippets_Misc/EF EDMX/MySample.Context.tt

```diff
@@ -0,0 +1,636 @@
+﻿<#@ template language="C#" debug="false" hostspecific="true"#>
```

Comment:
it is generated by the EF designer, it can be removed as everything we can do things manually, but why if VS can do that for me?